recirq/qml_lfe/circuit_blocks.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2020 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sycamore native circuit building blocks."""
+
+import os
+import cirq
+import cirq_google
+import functools
+
+
+@functools.lru_cache(maxsize=128)
+def _load_circuit(fname):
+    with open(os.path.join(os.path.dirname(__file__), fname), "r") as f:
+        return cirq.read_json(f)
+
+
+def tsym_block(qubits, params):
+    """Create a tsym block.
+
+    Y axis rotations followed by a real entried permutation operation.
+
+    Args:
+        qubits: The pair of qubits to apply this operation on.
+        params: The length 2 list of Y rotation angles.
+
+    Returns:
+        A list of `cirq.Operations` realizing a tsym block.
+    """
+    mapped_circuit = _load_circuit("tsym_permute.txt").transform_qubits(
```

Comment:
Why serialize these rather than have a generator function for them?  I am not sure I recommend one over the over, but I am curious to know the justification.

Reply:
It does take a little while O(seconds) to compute these gate sequences if you don't know them. I guess I could put a generator function in that just lists out the operations from the files. It feels a little less organized IMO, open to changing if you feel strongly.

Reply:
I feel strongly that either in this PR or in a subsequent one we include the code necessary to generate these data files